### ISSUE TITLE
Shining force 3 Scenario 1 - Ground texture shake

### ISSUE THREAD

**Issue by LuismaSP89** [FCare/Kronos#1430]
Some graphical issues on two very specific zones. (I'm playing this game and everything is fine except for this two zones)

First one is the loading menu screen, when you start the game in order to select a new game or load game options. With the fairy and the wizard. The background is totally messed.

Second one is on an specific town. The ground textures are totally broken. I think I managed to see the ground without issues one time but i'm not sure at 100%. To see this youy need to exit the building on the savestate provided. Or load the saved game attached also and exit the building too. You can see the different ways of ground corruption by entering/exit on a building.

![kronos 2023-01-28 12-46-25](https://user-images.githubusercontent.com/12205048/215265060-386c6672-dd88-47c2-8121-85fa5d40f74e.png)
![kronos 2023-01-28 12-43-19](https://user-images.githubusercontent.com/12205048/215265062-feda97d7-067e-4078-b4ab-67598bc2beda.png)
![kronos 2023-01-28 12-42-35](https://user-images.githubusercontent.com/12205048/215265063-6dc4f86e-69f0-419c-a9bb-5a02ee2caa10.png)
![kronos 2023-01-28 12-42-48](https://user-images.githubusercontent.com/12205048/215265064-ced129b8-4664-48ad-ad8e-8fc5a2ba5337.png)
![kronos 2023-01-28 12-44-52](https://user-images.githubusercontent.com/12205048/215265065-b59ea17f-309e-4ba4-b97f-7c0a41a4dc2e.png)
![kronos 2023-01-28 12-45-32](https://user-images.githubusercontent.com/12205048/215265067-2c651638-fe6e-4f81-82bd-cc3c61c4f1ec.png)

[Savestate & Saved file game.zip](https://github.com/FCare/Kronos/files/10527161/Savestate.Saved.file.game.zip)

**Comment by LuismaSP89** [comment]
Ok, after many tries, entering and exiting of the builds, I managed to see the correct texture of the floor. Please, use attached savestate to see it. I don't know what can be happening here. But loading this savestate will allow you to see it. So it must be something related or stored in the ram memory if I'm not mistaken.

[GS-9188_001.zip](https://github.com/FCare/Kronos/files/10527207/GS-9188_001.zip)
![kronos 2023-01-28 13-02-17](https://user-images.githubusercontent.com/12205048/215265566-aafaa4a1-a63a-4b74-8667-5426a0a556c2.png)

**Comment by BenjaminSiskoo** [comment]
CS mode ? With some filters ? :)

**Comment by LuismaSP89** [comment]
I always use OpenGL since the CS generates some graphical garbage in some games for me. And yes, the xbrz filter was enabled on the previous screenshots. 

Anyway, I disabled the filtering and tried with compute shaders, but seems the issue still remains : ( 

![kronos 2023-01-28 14-48-09](https://user-images.githubusercontent.com/12205048/215269931-39898a62-7b69-49cf-98e8-53eb0b307b05.png)
![kronos 2023-01-28 14-45-27](https://user-images.githubusercontent.com/12205048/215269935-e19324b3-ea0c-40f0-ae83-f03e4428b3b1.png)

**Comment by BenjaminSiskoo** [comment]
OpenGL is no longer maintained. Maybe some fixes will come later, but not sure.

**Comment by LuismaSP89** [comment]
Yes, I know that the CS is newer and with better compatibility. But for any reason the games looks worse compared with opengl at least in my case "for example the character's face in deep fear" 

In any case, seems this issue is happening for both backends, opengl & CS. So... not an video backend issue? Strange... let's see what Fcare thinks about this.

**Comment by BenjaminSiskoo** [comment]
@LuismaSP89 Kronos 2.3.1 Public or wip ?

**Comment by LuismaSP89** [comment]
Latest WIP, sorry for not providing that info before.

**Comment by LuismaSP89** [comment]
> @LuismaSP89 Kronos 2.3.1 Public or wip ?

Latest WIP, sorry for not providing that info before

**Comment by fafling** [comment]
This color problem has been fixed in january 25th WIP (0587ede). It was related to #1427 .
Tested with japanese redump image rev A.

**Comment by LuismaSP89** [comment]
> This color problem has been fixed in january 25th WIP ([0587ede](https://github.com/FCare/Kronos/commit/0587edef1498775317f0ad6b599a9c2fa8f6ab10)). It was related to #1427 . Tested with japanese redump image rev A.

Yes, you're right, I was using WIP from January 20th, not 25th, I didn't see there new PR's merged. I'm sorry.

In any case, that fix seems introduced another problem, now the ground texture is normal, but the ground is shaking, it's like it is alive and causes headache 

[2023-01-29 14-50-31.zip](https://github.com/FCare/Kronos/files/10529913/2023-01-29.14-50-31.zip)


See attached video.

PS: I also noticed some graphical corruption on the bios screen load on the bottom, it's a minor issue, but it started on this build from january 25th.

**Comment by LuismaSP89** [comment]
This should be the normal behaviour (Using again the build from january 20th)
[2023-01-29 15-01-27.zip](https://github.com/FCare/Kronos/files/10529930/2023-01-29.15-01-27.zip)

**Comment by fafling** [comment]
So now you're using the fbaccess branch WIP. This is a known issue on that branch with OpenGL only, there's a desynchronization between sprites and VDP2 graphics.

**Comment by LuismaSP89** [comment]
> So now you're using the fbaccess branch WIP. This is a known issue on that branch with OpenGL only, there's a desynchronization between sprites and VDP2 graphics.

Ok, I see. Let's wait then until the issue is fixed : ) thanks.

**Comment by LuismaSP89** [comment]
Update: Seems the fbaccess is not the only branch affected. The extui-align is also affected and maybe the 2.5.0 version when it's released.

**Comment by fafling** [comment]
Yes the issue is present in public v2.5.0.
Notice that OpenGL is no longer the default video core option. It has been switched to CS.

**Comment by LuismaSP89** [comment]
> Yes the issue is present in public v2.5.0.
> Notice that OpenGL is no longer the default video core option. It has been switched to CS.

Yes, I know that the CS is the most up-to-date video backend but in my case opengl looks better by far. The upscaling in CS is very bad. At least on my system, the games with 4x upscaling looks horrible with CS, specially in 3D games.
